saos-webapp/src/test/java/pl/edu/icm/saos/webapp/judgment/detail/correction/JudgmentCorrectionServiceTest.java

```diff
@@ -0,0 +1,194 @@
+package pl.edu.icm.saos.webapp.judgment.detail.correction;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import pl.edu.icm.saos.persistence.correction.JudgmentCorrectionRepository;
+import pl.edu.icm.saos.persistence.correction.model.CorrectedProperty;
+import pl.edu.icm.saos.persistence.correction.model.JudgmentCorrection;
+import pl.edu.icm.saos.persistence.correction.model.JudgmentCorrectionBuilder;
+import pl.edu.icm.saos.persistence.model.CommonCourtJudgment;
+import pl.edu.icm.saos.persistence.model.Judge;
+import pl.edu.icm.saos.persistence.model.Judgment;
+import pl.edu.icm.saos.persistence.model.SupremeCourtChamber;
+import pl.edu.icm.saos.persistence.model.SupremeCourtJudgment;
+import pl.edu.icm.saos.persistence.model.SupremeCourtJudgmentForm;
+
+import com.google.common.collect.Lists;
+
+@RunWith(MockitoJUnitRunner.class) 
+public class JudgmentCorrectionServiceTest {
+
+	
+	@Mock
+	private JudgmentCorrectionRepository judgmentCorrectionRepository;
+	
+	private JudgmentCorrectionService judgmentCorrectionService = new JudgmentCorrectionService();
+	
+	private Judgment ccJudgment = new CommonCourtJudgment();
+	private Judgment scJudgment = new SupremeCourtJudgment();
+	
+	
+	//------------------------ TESTS --------------------------
+	
+	@Test
+	public void findByJugmentId_CommonCourtJudgment() {
+		//given
+		int judgmentId = 123;
+		
+		List<JudgmentCorrection> unsortedJc = Lists.newArrayList();
+		unsortedJc.add(createJcJudgeDelete(ccJudgment));
+		unsortedJc.add(createJcCommonCourtJudgment());
+		
+		
+		List<JudgmentCorrection> expectedJc = Lists.newArrayList();
+		expectedJc.add(createJcCommonCourtJudgment());
+		expectedJc.add(createJcJudgeDelete(ccJudgment));
+		
+		when(judgmentCorrectionRepository.findAllByJudgmentId(judgmentId)).thenReturn(unsortedJc);
+		judgmentCorrectionService.setJudgmentCorrectionRepository(judgmentCorrectionRepository);
```

Comment:
It would be better to put this line into before method, so to not repeat it in each test case.